src/mixpanel/index.js

```diff
@@ -0,0 +1,25 @@
+import mixpanel from 'mixpanel-browser';
+
+const BROWSER_MIXPANEL_TOKEN = '9edede4b70de19f399736d5840872910';
+
+mixpanel.init(BROWSER_MIXPANEL_TOKEN);
+mixpanel.register({'timestamp': new Date().toString(), '$referrer': document.referrer})
+export const Mixpanel = {
```

Comment:
not sure why this is needed versus just `export const mixpanel`?  Is there plan to override some of these methods later?

Reply:
we need to do so since do not want to init everywhere

Reply:
you can still init in this module but re-export `mixpanel`

maybe I don't understand something but it's not clear why wrapper object is needed as well if the only goal is to init